packages/wdio-appium-service/src/launcher.js

```diff
@@ -39,7 +39,7 @@ export class AppiumLauncher {
 
     _startAppium(command, args, waitStartTime, callback) {
         log.debug(`Will spawn Appium process: ${command} ${args.join(' ')}`)
-        let process = spawn(command, args, { stdio: ['ignore', 'pipe', 'pipe'] })
+        let process = spawn('node', [command, ...args], { stdio: ['ignore', 'pipe', 'pipe'] })
```

Comment:
What happens if Appium is installed globally? This wouldn't work then anymore.

Reply:
AFAIK command is an absolute path to appium.
I just replaced it by PROJECT_DIR placeholder in example

Reply:
as you can see in 
```js
this.command = appiumConfig.command || this._getAppiumCommand()
```
the command can be either an absolute path to the launcher (as in your example) or just something like `appium` in case it is installed globally on the machine. For latter your patch would break.

Reply:
Path to appium executable is retrieved via [require.resolve](https://github.com/webdriverio/webdriverio/blob/6adfa2166f0656d4d61eb20c781b490374687e64/packages/wdio-appium-service/src/launcher.js#L79).

The point is that sole file name is not enough to spawn it on Windows.

Reply:
This should work when Appium is installed globally.
Unless somebody found a workaround for this problem by and used `appium: { command: 'node appium' }` in wdio.conf.js config

Reply:
Update: you are right. It's not possible to run global packages via node.

Reply:
Or even shorter but mixing concepts of command and args together
```js
async onPrepare(config) {
  if (appiumConfig.command) {
    this.commmand = appiumConfig.command
  } else {
    this.command = 'node'
    this.appiumArgs = this._getAppiumCommand()
  }

  this.appiumArgs.push(...this._cliArgsFromKeyValue(appiumConfig.args || {}))
}
```

Reply:
This should be resolved in https://github.com/webdriverio/webdriverio/pull/4136/commits/a49d7c8edc404bd1f5c4434136a023d54baae4b0